README.md

```diff
@@ -74,73 +53,96 @@ else:
 ![Generations](https://user-images.githubusercontent.com/56919667/261864112-0e419627-cbbe-4fb1-82e2-2637ee6392fb.png)
 
 The Imagine class acts as a facade, providing an interface to interact with all of our endpoints. It currently provides the following features:
-- **Text-To-Image**: generations() -> Response[ResponseImage]
-- **Image-Remix**: image_remix() -> Response[ResponseImage]
-- **Super-Resolution**: super_resolution -> Response[ResponseImage]
-- **Variate**: variate -> Response[ResponseImage]
-- **In-Painting**: in_painting() -> Response[ResponseImage]
-- **Alter-Image**: alter_image() -> Response[ResponseImage]
+- **Text-To-Image**: generations() -> Response[Image]
+- **Image-Remix**: image_remix() -> Response[Image]
+- **Super-Resolution**: super_resolution -> Response[Image]
+- **Variate**: variate -> Response[Image]
+- **In-Painting**: in_painting() -> Response[Image]
+- **Alter-Image**: alter_image() -> Response[Image]
```

Comment:
This is called guided generations, not alter image; Also mark this as not supported for now